static/loading-bar.js

```diff
@@ -0,0 +1,23 @@
+function LoadingBar(c){
+    this.meter = $('<span>', {style: 'width:0%;'});
+    this.elem = $('<div>', {style: 'display:none', class: 'meter animate '+c}).append(
+      this.meter.append($('<span>'))
+  );
+  $('body').prepend(this.elem);
+    return this;
+}
+
+LoadingBar.prototype.start = function(){
+    this.meter.css('width', '10%');
```

Comment:
I like that it starts off with _some_ width albeit I have seen it jump back on long pages (like _Technical Highlights_). Maybe we could make sure it doesn’t dip below 10 once it’s been started?